### ISSUE TITLE
<h1> is much more helpful for debugging than <title> for batch indexing

### ISSUE THREAD

**Issue by krispy1298** [pantheon-systems/solr-power#397]
Proposing a change in [this line](https://github.com/pantheon-systems/solr-power/blob/master/includes/class-solrpower-batch-index.php#L241).

Here is output from a CLI index with an error:
```
Solr HTTP error: OK (400)\n<html><head><title>Apache Tomcat\/7.0.68 - Error report<\/title><style><!--H1 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:22px;} H2 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:16px;} H3 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:14px;} BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;} B {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;} P {font-family:Tahoma,Arial,sans-serif;background:white;color:black;font-size:12px;}A {color : black;}A.name {color : black;}HR {color : #525D76;}--><\/style> <\/head><body><h1>HTTP Status 400 - ERROR: [doc=18807] unknown field 'valid_years'<\/h1><HR size="1" noshade="noshade"><p><b>type<\/b> Status report<\/p><p><b>message<\/b> <u>ERROR: [doc=18807] unknown field 'valid_years'<\/u><\/p><p><b>description<\/b> <u>The request sent by the client was syntactically incorrect.<\/u><\/p><HR size="1" noshade="noshade"><h3>Apache Tomcat\/7.0.68<\/h3><\/body><\/html>
```

`HTTP Status 400 - ERROR: [doc=18807] unknown field 'valid_years'` is much more helpful than `Apache Tomcat\/7.0.68 - Error report`.

**Comment by danielbachhuber** [comment]
Sorry for the late reply here, @krispy1298. This sounds like a good suggestion. In fact, I'd suggest parsing the `<h1>` if it exists, otherwise falling back to `<title>`

**Comment by danielbachhuber** [comment]
Fixed up in #407 407
